lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2649,6 +2649,114 @@ def calc_arrow(uvw, angle=15):
 
     quiver3D = quiver
 
+    def voxels(self, filled, color=None):
```

Comment:
There is no support here for specifying other properties such as edgecolors and edgewidth. Perhaps allow a **kwarg that can be any valid Poly3DCollection property?